app/web/src/components/AssetFuncAttachModal.vue

```diff
@@ -138,7 +138,6 @@ import { nilId } from "@/utils/nilId";
 import CodeEditor from "./CodeEditor.vue";
 
 const props = defineProps<{
-  schemaVariantId?: string;
   assetId?: string;
 }>();
```

Comment:
Let's only take in the one prop we need since the `schemaVariantId` will change alongside it.